aspnetcore/fundamentals/ihostedservice.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Background tasks with IHostedService in ASP.NET Core
+author: guardrex
+description: Learn how to implement background tasks with IHostedService in ASP.NET Core.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 02/12/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/ihostedservice
+---
+# Background tasks with IHostedService in ASP.NET Core
```

Comment:
I suggest replacing "IHostedService" in the title with "hosted services". Using "hosted services" or something that isn't a cryptic API name seems a bit more inviting.